app/views/events/index_user.html.erb

```diff
@@ -20,12 +29,13 @@
           <td><%= event.mandatory %></td>
           <td>
             <%= link_to 'Details', event_path(event.id),  class: "btn btn-secondary" %>
-            <% if event.date.utc+@utc_offset <= Time.now.utc && Time.now.utc <= event.end_time.utc+@utc_offset %>
+            <% if event.date.utc+@utc_offset <= Time.now.utc && Time.now.utc <= event.end_time.utc+@utc_offset and not @user_events.include?(event) %>
```

Comment:
If we could get this same condition in the index_admin page so that the sign in button also hides for admins that'd be awesome.